python/lsst/ts/salobj/topics/controller_command.py

```diff
@@ -172,6 +176,36 @@ async def _run_callback(self, data):
         data : `DataType`
             Command data.
         """
+        if self.authorize:
+            try:
+                identity = data.private_identity
+                auth_error = None
+                if "@" in identity:
+                    # A user
+                    if (
+                        identity != self.salinfo.domain.user_host
+                        and identity not in self.salinfo.authorized_users
+                    ):
+                        auth_error = (
+                            f"User identity {identity!r} "
+                            f"not self ({self.salinfo.domain.user_host}) "
+                            f"and not in {self.salinfo.authorized_users}"
+                        )
+                elif identity in self.salinfo.non_authorized_cscs:
+                    auth_error = f"CSC identity {identity!r} in {self.salinfo.non_authorized_cscs}"
+                if auth_error:
```

Comment:
`auth_error = None` so I think you want this `if` to be like:

```
if auth_error is None:
```

Reply:
I'll change it. I'm a bit torn. It is impossible for the auth error to be an empty str, so the code works as is, and it would be a bad thing to raise this error with an empty string, but I'm not keen to initialize auth_error to "" so I'll change it.